templates/bake/Model/table.twig

```diff
@@ -33,8 +36,6 @@ class {{ name }}Table extends Table
      */
     public function initialize(array $config): void
     {
-        parent::initialize($config);
```

Comment:
Why is this parent call moved down?

Reply:
with the intention that code from persistent class is always overwritten. so user can override code from main class also when he runs bake model again initialize code will overwrite generated code if needed.